main.java

```diff
@@ -0,0 +1,100 @@
+        System.out.println("Задача 1");
+//        Объявите переменные типа int, byte, short, long, float, double.
+//        Название переменных может быть любым, но если состоит из двух слов и более, должно соответствовать правилу camelCase.
+//        Выведите в консоль значение каждой переменной в формате «Значение переменной … с типом … равно …».
+        int task1Int=2147483647;
+        System.out.println("Значение переменной task1Int с типом int равно "+task1Int);
+        byte task1Byte=-128;
+        System.out.println("Значение переменной task1Byte с типом byte равно "+task1Byte);
+        short task1Short=-32767;
+        System.out.println("Значение переменной task1Short с типом Short равно "+task1Short);
+        long task1Long=-9223372036854775808L;
+        System.out.println("Значение переменной task1Long с типом long равно "+task1Long);
+        float task1Float=689.4F;
+        System.out.println("Значение переменной task1Float с типом float равно "+task1Float);
+        double task1Double=-1111111111.11111111111;
+        System.out.println("Значение переменной task1Double с типом double равно "+task1Double);
+//        int str = 256;
+//        System.out.println(str.getClass().getSimpleName()); // => String
+        System.out.println("Задача 2");
+//        Ниже дан список различных значений.
+//        Инициализируйте переменные, используйте изученные ранее типы переменных.
+        float task2Float =27.12f;
+        System.out.println("Значению "+task2Float+" присвоен тип float");
+        long task2Long =987678965549L;
+        System.out.println("Значению "+task2Long+" присвоен тип long");
+        float task2Float1 =2.786F;
+        System.out.println("Значению "+task2Float1+" присвоен тип float");
+        int task2Int =569;
+        System.out.println("Значению "+task2Int+" присвоен тип int");
+        int task2Int1 =-159;
+        System.out.println("Значению "+task2Int1+" присвоен тип int");
+        short task2short =27897;
+        System.out.println("Значению "+task2short+" присвоен тип short");
+        byte task2Byte =67;
+        System.out.println("Значению "+task2Byte+" присвоен тип Byte");
+        System.out.println("Задача 3");
+//  Три школьных учителя, Людмила Павловна, Анна Сергеевна и Екатерина Андреевна, ведут три класса.
+//  У Людмилы Павловны — 23 ученика , у Анны Сергеевны — 27 учеников, у Екатерины Андреевны — 30 учеников.
+//  Три учительницы закупили все вместе 480 листов бумаги на все три класса.
+//  Посчитайте, сколько достанется листов каждому ученику.
+//  Результат задачи выведите в консоль в формате: «На каждого ученика рассчитано … листов бумаги».
+        int pupilLP=23;    //  Людмилы Павловны — 23 ученика
+        int pupilAS=27;    //  Анны Сергеевны — 27 учеников
+        int pupilEA=30;    //  Екатерины Андреевны — 30 учеников
+        int allList=480;  //  закупили все вместе 480 листов бумаги
+        int List=allList/(pupilLP+pupilAS+pupilEA);
```

Comment:
```suggestion
        int list=allList/(pupilLP+pupilAS+pupilEA);
```
названия переменных всегда пишем с маленькой буквы